lib/charms/operator_libs_linux/v0/systemd.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2021 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Abstractions for stopping, starting and managing system services via systemd.
+
+This library assumes that your charm is running on a platform that uses systemd. E.g.,
+Centos 7 or later, Ubuntu Xenial (18.04) or later.
```

Comment:
I think Xenial had support systemd in which case, the version number would need to be fixed (18.04 -> 16.04).

Reply:
Whoops! You're right, Xenial's version should be 16.04. I should remember that, because I started working at Canonical right after Xenial released :-)